.circleci/config.yml

```diff
@@ -239,74 +231,166 @@ jobs:
-          packages: "./ddtrace/..."
+          name: test_aws
+          module: ./contrib/aws/aws-sdk-go/aws
       - go_test:
-          name: test_elasticsearch
-          executor: go_elasticsearch
-          packages: "`go list ./contrib/... | grep 'elastic' | xargs`"
+          name: test_memcached
+          executor: go_memcached
+          module: ./contrib/bradfitz/gomemcache/memcache
           after_deps:
             - dockerize_wait:
-                address: http://localhost:9200
-            - dockerize_wait:
-                address: http://localhost:9201
+                address: tcp://localhost:11211
       - go_test:
-          name: test_kafka_zookeeper
+          name: test_confluent-kafka-go
           executor: go_kafka_zookeeper
-          packages: "`go list ./contrib/... | grep -P 'Shopify|kafka' | xargs`"
+          module: ./contrib/confluentinc/confluent-kafka-go/kafka
           after_deps:
             - dockerize_wait:
                 address: tcp://localhost:2181
             - dockerize_wait:
                 address: tcp://localhost:9092
       - go_test:
-          name: test_memcached
-          executor: go_memcached
-          packages: "`go list ./contrib/... | grep 'memcache' | xargs`"
+          name: test_sql
+          executor: go_sql
+          module: ./contrib/database/sql
           after_deps:
             - dockerize_wait:
-                address: tcp://localhost:11211
+                address: tcp://localhost:3306
+            - dockerize_wait:
+                address: tcp://localhost:5432
       - go_test:
-          name: test_mongo
+          name: test_go-restful
+          module: ./contrib/emicklei/go-restful
+      - go_test:
+          name: test_redigo_old
+          executor: go_redis
+          module: ./contrib/garyburd/redigo
+          after_deps:
+            - dockerize_wait:
+                address: tcp://localhost:6379
+      - go_test:
+          name: test_gin
+          module: ./contrib/gin-gonic/gin
+      - go_test:
+          name: test_mongo_globalsign
           executor: go_mongo
-          packages: "`go list ./contrib/... | grep -P 'globalsign|mongo' | xargs`"
+          module: ./contrib/globalsign/mgo
           after_deps:
             - dockerize_wait:
                 address: tcp://localhost:27017
+      - go_test:
+          name: test_chi
+          module: ./contrib/go-chi/chi
       - go_test:
           name: test_redis
           executor: go_redis
-          packages: "`go list ./contrib/... | grep -P 'redis|redigo' | xargs`"
+          module: ./contrib/go-redis/redis
+          after_deps:
+            - dockerize_wait:
+                address: tcp://localhost:6379
+      # - go_test:
+      #    name: test_cassandra
+      #    executor: go_cassandra
+      #    module: ./contrib/gocql/gocql
+      #    after_deps:
+      #      - dockerize_wait:
+      #          address: tcp://localhost:9042
+      #          timeout: 2m
```

Comment:
Do we need this ^ ?

Reply:
At least it visualizes that it is not tested in CircleCI... I can try what happens if I enable it back,